### PR TITLE
Update changelog with latest bug fixes.

### PR DESCRIPTION
**Pull request by vickycl**
## Purpose of this PR

I've gone through all the changes that have landed and added to the changelog any that should be called out as a bug fix.

**Ticket/Jira #:**

## Testing

**Functional Testing status:**

**Performance Testing status:**

## Overall Product Risks

**Complexity:**

**Halo Effect:**

## Additional information

**Note to reviewers:**

**Reminder:**

- [x] Add entry in CHANGELOG.md _(if applicable)_

### CODE REVIEW

File: package/com.unity.formats.usd/CHANGELOG.md [hunk 1]
```diff
@@ -2,14 +2,24 @@
...
+- Fixed a bug on export where SkelRoot and Skeleton Prim were missing class identifiers.
+- Fixed 'ExportSelectedWithChildren' on Mac.
+- Fixed texture mapping for exported meshes.
+- Fixed  a bug causing PointInstances to be duplicated.
+- Fixed regression in JointsMatch().
+- Fixed scene not being marked dirty after changing load payload flag.
+- Fixed exported Prims not having a type and fixed export for objects with same name.
 - Fixed a bug causing an ApplicationException "all sample times in times and baseTime must either all be numeric or all be default" to be thrown when refreshing a PointInstancer.
-- Optimized triangulation to reduce the chance of a "Timed out while waiting for thread" error on importing a complex mesh.
+- Fixed an import bug causing instanced primitives not to be sanitized.
```
Comment (michaeljblain) [inline package/com.unity.formats.usd/CHANGELOG.md]: We should perhaps briefly define what sanitized means - or is it defined elsewhere?

**Review by michaeljblain — CHANGES_REQUESTED** [review]
One minor suggestion, otherwise looks good.

File: package/com.unity.formats.usd/CHANGELOG.md (hunk 1 above)
Comment (vickycl) [reply package/com.unity.formats.usd/CHANGELOG.md]: I'll add more detail here, but I think we should define this in the documentation as hidden in the changelog possibly isn't the place for it- I'll add a full explanation of the sanitisation to the new docs I'm preparing.

Comment (michaeljblain) [reply package/com.unity.formats.usd/CHANGELOG.md]: Sounds good.

**Review by michaeljblain — APPROVED** [review]

**Pull request merged**